WS2012R2/lisa/xml/STOR_VHDX.xml

```diff
@@ -114,6 +116,7 @@
                 <param>DefaultSize=64TB</param>
                 <!-- <param>Path=G:\VHDs</param> -->
                 <param>ControllerType=SCSI</param>
+                <param>SCSI=0,0,Dynamic,512,64TB</param>
```

Comment:
why do we require this line if all the other params have been moved outside of it and individually?